models/Sequence.model.js

```diff
@@ -0,0 +1,16 @@
+const { Schema, model } = require("mongoose");
+
+const sequenceSchema = new Schema(
+  {
+    notes: [String],
+    coordinates: [String],
```

Comment:
I think we will need one more array for coordinates. More info here: https://github.com/ivvva/sequencer-2/pull/6 

I suggest that instead of one array called coordinates, we do two arrays: first one called `drawingX` and second one  `drawingY`. lmk if you come up with a better idea for naming.